engine/input/src/input.cpp

```diff
@@ -479,11 +498,11 @@ namespace dmInput
         action->m_HasGamepadPacket = 0;
     }
 
-    struct UpdateContext
+    struct UpdateActionContext
```

Comment:
Making it clearer that it's about the actions/binding, and not the input context